tests/unit/parsers/test.fluent.js

```diff
@@ -28,13 +28,13 @@ choose-download-folder-title =
     const addonLinter = new Linter({ _: ['bar'] });
     const parser = new FluentParser(
       `
-key67
+key67 =
     .label = Sign In To &syncBrand.shortName.label;…
     .accesskey = Y
-key68
+key68 =
     .label = Sync Now
     .accesskey = S
-key69
+key69 =
```

Comment:
Changes in the actual spec, https://github.com/projectfluent/fluent/blob/master/spec/CHANGELOG.md